ldap/servers/plugins/retrocl/retrocl_po.c

```diff
@@ -691,6 +691,12 @@ retrocl_postob(Slapi_PBlock *pb, int optype)
         (void)slapi_pblock_get(pb, SLAPI_ENTRY_POST_OP, &post_entry);
         break;
     }
+    if (entry == NULL) {
+        /* To avoid coverity scan warning, but in practice, it should never happen. */
+        slapi_log_err(SLAPI_LOG_ERR, RETROCL_PLUGIN_NAME,
+                      "retrocl_postob - Operation on NULL entry\n", rc);
+        return 0;
```

Comment:
So the operation is not logged in retroCL. Shouldn't it return an error (LDAP_OPERATIONS_ERROR ?)

Reply:
I do not think it really matters, 
  Only abandon and extended operations (that are not logged in the retrocl anyway) does have a target entry)
  So the test is always false unless there is an heap/memory corruption.
    ( and frankly, at this point what we do does not matter much as the instance will probably crash anyway)
BTW if you really want to retuirn an error, LDAP_UNWILLING_TO_PERFORM will be better. Returning LDAP_OPERATIONS_ERROR in such case is an abuse. (LDAP_OPERATIONS_ERROR (i.e ldap operationms sequence erro) should only be returned by bind operation)